website/docs/tutorials/06-deploying/02-migrations/invoke.md

```diff
@@ -0,0 +1,75 @@
+# `invoke`
+
+```python
+def invoke(
+    contract_address: int,
+    function_name: str,
+    inputs: list[int] | dict[str, Any] | None = None,
+    max_fee: Optional[int] = None, # in Wei
```

Comment:
```suggestion
    inputs: list[int] | dict[str, Any] | None = None,
    max_fee: int | None = None, # in Wei
```

There should be one way how to handle optional.

Reply:
Fixed